### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by nrdevau**
## Purpose of this pull request

This pull request (PR) adds Module Enable step for PWA modules. Corrected search syntax for Magento_*Pwa modules.

PWA Studio looks to work ootb when also building the M2 backend, but in my use case, I have an existing M2 backend and it's been a little tricky getting things working. One reason is because my backend didn't have these PWA modules installed (or enabled) and I wanted to clear it up in the docs. Ideally the PWA docs would also make it clear that if using an existing backend, some additional modules are required

## Affected pages

- https://developer.adobe.com/commerce/pwa-studio/metapackages/open-source/

### CODE REVIEW

**Comment by nrdevau** [comment]
I have since signed the CLA, what more do I do?

File: src/pages/metapackages/open-source/index.md
Comment (nrdevau) [inline src/pages/metapackages/open-source/index.md:59]: whoever is reviewing this, we need to decide whether we want to just generalise which modules will need to be enabled or list each one. The benefit of listing each one is it becomes abundantly clear which modules the composer magento/pwa package offers. The downside is maintainability because whenever a new module is added to that package this becomes outdated and needs to be updated. 

Ideally we'd dynamically build the list somewhere for external devs to know what to expect in that package without having to download the manifest or something

**Comment by jeff-matthews** [comment]
Thanks @nrdevau. 

Validation test is running. Assuming no errors, a repo maintainer will review soon and provide feedback if necessary. After it's approved, it'll get merged and deployed.

File: src/pages/metapackages/open-source/index.md
Comment (jeff-matthews) [reply src/pages/metapackages/open-source/index.md:59]: I'm in favor of listing each one. That seems more helpful.

As far as I know, no new modules are planned and manually adding all current modules to this page for reference is relatively low risk. If that changes, we can always revisit and look into automation options.

On an unrelated note, you'll need to fix the .md linting errors to pass the validation check. The code block spacing needs some adjustment (add line between block and preceding step. Possibly reduce indentation of block by one space.

File: src/pages/metapackages/open-source/index.md
Comment (jeff-matthews) [inline src/pages/metapackages/open-source/index.md:61]:
```suggestion
At this point, you should see that all the PWA modules are now enabled.
```

File: src/pages/metapackages/open-source/index.md
Comment (jeff-matthews) [inline src/pages/metapackages/open-source/index.md:53]: I know you didn't change this line, but the guidance to ensure that no modules are "_enabled_" seems odd... Is it possible that this is a mistake and should say "ensure no modules are _disabled_...". that would make more sense, especially with the following sentence linking to info for _enabling_ modules. And that last sentence seems obsolete now with your addition of step 2.

File: src/pages/metapackages/open-source/index.md
Comment (jeff-matthews) [inline src/pages/metapackages/open-source/index.md:55]:
```suggestion
1. Enable the PWA modules
```

Comment (jeff-matthews) [reply src/pages/metapackages/open-source/index.md:55]: All ordered lists should use `1.` in .md. They'll render in order during the build.

File: src/pages/metapackages/open-source/index.md
Comment (nrdevau) [reply src/pages/metapackages/open-source/index.md:53]: Yeah I thought it was odd, but on one hand you might want to expect the modules are first _disabled_ in order to then justify enabling them, but the fact that they are disabled would be the case 99% of the time this is added. The only time they wouldn't be is if the `app/etc/config.php` already had those modules as enabled (which would mean you can skip step 2 entirely)

I'm happy to change it and remove the last line (linking to how to do it) in place of introducing the step 2 as I have got currently. I think the semantics at this point would assume someone knows the `add a module and enable it` flow in M2.

The other option is we remove the new step 2 and just have some sort of "just so you know here are you modules you should expect to see" to list the modules, then we can keep the internal link (which helps with SEO if nothing else)

File: src/pages/metapackages/commerce/index.md
Comment (nrdevau) [inline src/pages/metapackages/commerce/index.md:65]: I can't install the magento/pwa-commerce package, so this list will have to be provided :)

File: src/pages/metapackages/commerce/index.md
Comment (nrdevau) [inline src/pages/metapackages/commerce/index.md:1]: The steps are common for the most part between open-source and commerce and might make sense to use the _includes for partials, if extending this in future.

File: src/pages/metapackages/commerce/index.md
Comment (jeff-matthews) [inline src/pages/metapackages/commerce/index.md:57]:
```suggestion
The following modules should now be enabled:
```

File: src/pages/metapackages/commerce/index.md
Comment (jeff-matthews) [inline src/pages/metapackages/commerce/index.md:67]: Let's delete this line since it's similar to line 57.

File: src/pages/metapackages/open-source/index.md
Comment (jeff-matthews) [inline src/pages/metapackages/open-source/index.md:55]:
```suggestion
The following modules should be enabled:
```

File: src/pages/metapackages/open-source/index.md
Comment (jeff-matthews) [inline src/pages/metapackages/open-source/index.md:64]: Let's delete this line since it's similar to line 55.

File: src/pages/metapackages/commerce/index.md
Comment (jeff-matthews) [inline src/pages/metapackages/commerce/index.md:59]: Remove indentation and wrap in code fences with bash language specified after first fence

```
```bash
```

File: src/pages/metapackages/open-source/index.md
Comment (jeff-matthews) [inline src/pages/metapackages/open-source/index.md:57]: Remove indentation and wrap in code fences with bash language specified after first fence

```
```bash
```

**Review by jeff-matthews — COMMENTED** [review]
Nicely done @nrdevau! Thank you!

Good idea re: _includes in the future.

I'll work on getting the list of modules for the commerce metapackage. I should have that by tomorrow.

File: src/pages/metapackages/commerce/index.md
Comment (keharper) [reply src/pages/metapackages/commerce/index.md:59]: I think `terminal` is a better code fence. This is command output.

File: src/pages/metapackages/commerce/index.md
Comment (jeff-matthews) [reply src/pages/metapackages/commerce/index.md:65]: @nrdevau 

The only other module for commerce is `Magento_ContactGraphQlPwa`.

**Comment by keharper** [comment]
I'm leaving a few comments because I saw GraphQL being mentioned.

Just as an FYI, the GraphQL coverage has been ported to core Open Source as of 2.4.7-beta2, if you're interested.

Do you also need to ensure the modules that end with *Aux are enabled?

There's only one module within the `magento2-pwa-commerce` repo: BannerGraphQlAux. I don't know if that's worth mentioning.

**Comment by jeff-matthews** [comment]
Good question @keharper. I'm not sure about the aux extensions.

File: src/pages/metapackages/commerce/index.md
Comment (jeff-matthews) [reply src/pages/metapackages/commerce/index.md:59]: Good cal @keharper. I didn't think "terminal" was supported, but it is.

**Comment by nrdevau** [comment]
I think the actual issue I had was after installing the venia sample data. For some reason that removed the metapackage. I spoke too soon. I have now rectified. Sorry for the confusion. I've been out of M2 dev for a couple years so there's a lot to get familiarised with again :sweat_smile:

**Comment by nrdevau** [comment]
> Just as an FYI, the GraphQL coverage has been ported to core Open Source as of 2.4.7-beta2, if you're interested.
@keharper are you saying that this entire document will be redundant when that's released? Will all the PWA and AUX modules be bundles into the core? 

I haven't followed the pulse w/r/t how much support PWA's will have OOTB in the Magento backend. More just curious to know moving forward :)

File: src/pages/metapackages/commerce/index.md
Comment (jeff-matthews) [inline src/pages/metapackages/commerce/index.md:55]:
```suggestion
Make sure that all of the required modules are enabled when you run `bin/magento module:status`. See, [Enable or disable a component](https://developer.adobe.com/commerce/php/development/build/component-management/) in the _PHP Developer Guide_ for instructions.
```

File: src/pages/metapackages/open-source/index.md
Comment (jeff-matthews) [inline src/pages/metapackages/open-source/index.md:53]:
```suggestion
Make sure that all of the required modules are enabled when you run `bin/magento module:status`. See, [Enable or disable a component](https://developer.adobe.com/commerce/php/development/build/component-management/) in the _PHP Developer Guide_ for instructions.
```

**Review by jeff-matthews — COMMENTED** [review]
Almost there @nrdevau. Just a couple minor editorial suggestions and then I think we can get this merged.

@keharper, do you have any more suggestions/questions?

**Comment by keharper** [comment]
@nrdevau Yeah, the magento2-pwa repo is pretty much ending its end of life, especially from a GraphQL point a view. (I don't know if the repo provides non-GraphQL functionality.)

**Comment by nrdevau** [comment]
@jeff-matthews / @keharper in light of that information around EOL of this metapackage, should we add some sort of sunset message at the top of each of these metapackage pages while we're here?

I will modify as suggested and maybe make an issue to discuss the EOL messaging separately to this little change

**Review by jeff-matthews — APPROVED** [review]
@nrdevau

No, I think there's only been unconfirmed _speculation_ of EOL in this thread. It's up to Adobe and the PWA Studio project maintainers to make that call. I think we should just stick with the changes from your latest commit.

**Pull request merged**
